### ISSUE TITLE
Will be great to have slider to interact with numbers and color pickers.

### ISSUE THREAD

**Issue by nitanagdeote** [vizhub-core/vzcode#591]
![image](https://github.com/vizhub-core/vzcode/assets/13540669/e3c35033-bb42-4a03-9467-ff061cc04dfa)

**Comment by curran** [comment]
I agree! This old tool called Inlet by @enjalot has this feature: https://enjalot.github.io/Inlet/

![image](https://github.com/vizhub-core/vzcode/assets/68416/73ab1be9-5cc9-4bed-abe2-005dd3afe3ad)

I would be open to getting that working in VZCode, for sure! It would require a major upgrade of Inlet, or a re-write of something similar, that works with CodeMirror 6 (Inlet is based on CodeMirror 5, so we can't use it directly).

**Comment by nitanagdeote** [comment]
thas exactly what we need
